_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,29 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Enabled HTTP/2 on Apache
+
+On all platforms HTTP/2 is now enabled.
```

Comment:
```suggestion
HTTP/2 is now enabled by default on all platforms.
```
Feels more natural to me, but then you have two sentences that start with `HTTP/2` so :woman_shrugging: